packages/component-library-react/src/Accordion.tsx

```diff
@@ -115,7 +115,9 @@ export const AccordionSection = forwardRef(
             {children}
           </section>
         ) : (
-          <div {...panelAttributes}>{children}</div>
```

Comment:
I wonder if `aria-labelledby={buttonId}` works in this case, because I think you can only label things with a (implied) `role`.

Reply:
The `section` role is optional to avoid "landmark overload", so no label for the div is intentional in this case. I don't think this needs fixing.

Adding `id={panelId}` is a good improvement of course.

Reply:
I also believe the aria-labelledby can go. 
It could indeed lead to a very busy landmark list rotor.